crypto/rand/rand_lib.c

```diff
@@ -46,7 +46,7 @@ void rand_read_tsc(RAND_poll_fn cb, void *arg)
     unsigned char c;
     int i;
 
-    for (i = 0; i < 10; i++) {
+    for (i = 0; i < TSC_READ_COUNT; i++) {
         c = (unsigned char)(OPENSSL_rdtsc() & 0xFF);
```

Comment:
It should be `OPENSSL_ia32cap_P[2]` in line 67, because pod file refers to bit as 64+18 and index in array is bit position divided by 32. [Originally I told you 64, but then immediately corrected myself, which you actually acknowledged.]